xc/xenops_server_xen.ml

```diff
@@ -749,6 +749,9 @@ module HOST = struct
          features_pv.(3)  <- Int64.logor features_pv.(3)  0x2L;
          features_hvm.(3) <- Int64.logor features_hvm.(3) 0x2L;
 
+         (* Set X86_FEATURE_IBS in e1c for HVM guests *)
+         features_hvm.(3) <- Int64.logor features_hvm.(3) 0x400L;
```

Comment:
Just a note: `0x400L` can be also expressed as `0b100_0000_0000L` if that were more natural.

Reply:
In OCaml you can do:

```
let (<<) = Int64.shift_left in
1L << 11
```